.github/workflows/ci.yml

```diff
@@ -4,7 +4,11 @@
 
 name: 💡🏠
 
-on: [pull_request]
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
any reason to only run with PRs against master? PRs against PRs happen occasionally

Reply:
not a particularly good reason just that the syntax for it in github actions is confusing :) so sure let's do it

Reply:
> just that the syntax for it in github actions is confusing

me having to look up the syntax is why I didn't approve right away :)